.github/jobs/bash_functions.sh

```diff
@@ -25,3 +25,13 @@ function time_command {
   fi
   return $error
 }
+
+# utility function to construct the DockerHub tag name to be used,
+# replacing slashes with underscores
+
+function get_dockerhub_tag {
+
+  tag_name=`echo ${DOCKERHUB_REPO}:${SOURCE_BRANCH} | sed 's%/%_%g'`
```

Comment:
```suggestion
  echo ${DOCKERHUB_REPO}:${SOURCE_BRANCH} | sed 's%/%_%g';
```
You can't return a string in a bash function, so just echo the value